### ISSUE TITLE
CLI arguments are ignored

### ISSUE THREAD

**Issue by Artusamak** [Happyculture/combawa#6]
If I have a settings set in my .env and i pass an argument to override it, it's value is ignored.

Example: Set the setting to fetch the dump from prod to 1 by default in settings.sh or .env and run `combawa.sh -f 0`, the build will be run with the value of 1 to the fetch flag.

That's annoying! CLI args should be the value taken care as last possible override.

Settings.sh > .env > CLI args is the correct order i think.

**Comment by DuaelFr** [comment]
thank you for your report!
I cannot reproduce this issue with the latest version, though.

```
 $ ./vendor/bin/combawa.sh
...
[Environment built] dev

 $ ./vendor/bin/combawa.sh -e prod
...
[Environment built] prod
```

Would you provide steps to reproduce, please? ;)

**Comment by Artusamak** [comment]
> Example: Set the setting to fetch the dump from prod to 1 by default in settings.sh or .env and run `combawa.sh -f 0`, the build will be run with the value of 1 to the fetch flag.

It's described in the initial issue =) It may be related to #4 though.

**Comment by Artusamak** [comment]
I confirm that the fix for #4 fixed this issue.
